src/app/space/forge-wizard/components/forge-exception/forge-exception.component.less

```diff
@@ -0,0 +1,146 @@
+@import (reference) '../../../../../assets/stylesheets/shared/osio.less';
+@icon-path: '../../../../../assets/stylesheets/shared/_icons.less';
+.close-button {
+  text-align: right;
+  margin-bottom: 2em;
+}
+.app-generator-message {
+  @color-success: @color-pf-green;
+  @color-success-border: @color-success;
+  @color-success-background: @color-pf-black-200;
+  @color-error: @color-pf-red;
+  @color-error-border: @color-error;
+  @color-error-background: @color-pf-black-100;
+  @color-warning: @color-pf-gold;
+  @color-warning-border: @color-warning;
+  @color-warning-background: @color-pf-black-100;
+  @standard-spacing: @grid-gutter-width/2;
+  padding: @standard-spacing;
+  position: relative;
+  width: 100%;
+  display: block;
+  z-index: 1;
+  &.hide { display: none; }
+  .message-title-container {
```

Comment:
There are quite a few nested styles in this file; BEM naming should be used and nesting removed to avoid specificity problems.